crs_motion_planning/src/path_planning_utils.cpp

```diff
@@ -844,22 +865,28 @@ bool crsMotionPlanner::generateFreespacePlan(const tesseract_motion_planners::Jo
     {
       return false;
     }
-
-    joint_trajectory.header.frame_id = config_->world_frame;
-    // Modify time
-    if (config_->use_gazebo_sim_timing)
+  }
+  else
+  {
+    joint_trajectory = ompl_joint_traj;
+  }
+  joint_trajectory.header.frame_id = config_->world_frame;
+  // Modify time
+  if (config_->use_gazebo_sim_timing)
+  {
+    RCLCPP_INFO(logger_, "MODIFYING TRAJECTORY TIME OUTPUT");
+    for (int i = 0; i < joint_trajectory.points.size(); ++i)
     {
-      RCLCPP_INFO(logger_, "MODIFYING TRAJECTORY TIME OUTPUT");
-      for (int i = 0; i < joint_trajectory.points.size(); ++i)
-      {
-        joint_trajectory.points[static_cast<size_t>(i)].time_from_start.sec = 0;
-        joint_trajectory.points[static_cast<size_t>(i)].time_from_start.nanosec = 1e8;
-      }
+      joint_trajectory.points[static_cast<size_t>(i)].time_from_start.sec = 0;
+      joint_trajectory.points[static_cast<size_t>(i)].time_from_start.nanosec = 1e8;
```

Comment:
So each point has a  0.1 timestamp?

Reply:
Yes, this is if you are using the old gazebo trajectory plugin, that way each step is 0.1 seconds apart and no real time parameterization is performed